ground/src/main/java/com/google/android/ground/ui/datacollection/tasks/multiplechoice/MultipleChoiceTaskViewModel.kt

```diff
@@ -95,7 +95,13 @@ class MultipleChoiceTaskViewModel @Inject constructor() : AbstractTaskViewModel(
         task.multipleChoice,
         selectedIds.map {
           // if the other option is selected grab the associated text save instead of otherId
-          if (it == OTHER_ID) OTHER_PREFIX + otherText.trim() + OTHER_SUFFIX else it
+          if (it == OTHER_ID) {
+            OTHER_PREFIX +
+              (if (otherText.isNotBlank()) otherText.trim() else "Other") +
+              OTHER_SUFFIX
+          } else {
+            it
+          }
```

Comment:
Can we revert this?

Reply:
but without this the value will always empty, we have to show "Other" text, right?
Or it will be taken care by backend?

Reply:
We don't need a default value for the other text.

Reply:
done